cmd/spiffe-helper/config/config.go

```diff
@@ -46,28 +48,36 @@ type JWTConfig struct {
 	JWTSVIDFilename string `hcl:"jwt_svid_file_name"`
 }
 
+func ParseFlags() (string, bool) {
+	configFile := flag.String("config", "helper.conf", "<configFile> Configuration file path")
+	daemonModeFlag := flag.Bool(daemonModeFlagName, true, "Exit once the requested objects are retrieved")
+	flag.Parse()
```

Comment:
why to parse here? generally it is expected than flags are parsed in main so it is easy to found,
is it for testing?

Reply:
Yes for unit testing. See related tested here: https://github.com/spiffe/spiffe-helper/pull/161/files#diff-a7f12ddbf036810a09ab419309d276dae4b679371fc962c1ad5a63cd0cf790ccR400

Reply:
i can move back to main and have duplicate for tests if you think thats better.

Reply:
that will be great

Reply:
done